solarwinds_apm/configurator.py

```diff
@@ -341,24 +352,16 @@ def _configure_metrics_exporter(
         self,
         apm_config: SolarWindsApmConfig,
     ) -> None:
-        """Configure SolarWinds OTel metrics exporters, environment
-        configured if any, or none if agent disabled."""
-        if not apm_config.agent_enabled:
-            logger.error("Tracing disabled. Cannot set metrics exporter.")
-            return
```

Comment:
I've removed this check because the caller is already doing the enabled check (`_configure_otel_components` further up)